pkg/manager/daemon_adaptor.go

```diff
@@ -146,7 +146,8 @@ func (m *Manager) BuildDaemonCommand(d *daemon.Daemon, bin string, upgrade bool)
 			}
 
 			cmdOpts = append(cmdOpts,
-				command.WithConfig(d.ConfigFile("")),
+				command.WithConfig(d.States.ID),
+				command.WithConfigSource(""),
```

Comment:
Expose `ConfigSource` as a configuration entry? Otherwise, use `/run/containerd-nydus/system.sock` as an argument for this method.

Reply:
> Expose `ConfigSource` as a configuration entry? Otherwise, use `/run/containerd-nydus/system.sock` as an argument for this method.

I am also considering this point. This value is a property of the Controller class in system. go. To obtain this socket path, you should first obtain the Controller instance. I have no idea how to obtain this instance yet.